packages/terra-application-links/src/tabs/ApplicationTabMenuContent.scss

```diff
@@ -0,0 +1,7 @@
+:local {
+  .list {
+    background-color: #fff;
```

Comment:
Should the background use white or transparent?

Reply:
I think we also need to ensure all of our colors are themeable as well.